admin/package-files/docker-plugin/upstart/flocker-docker-plugin.conf

```diff
@@ -0,0 +1,11 @@
+# flocker-docker-plugin - flocker-docker-plugin job file
+
+description "Flocker Docker Plugin"
+author "ClusterHQ <support@clusterhq.com>"
+
+start on starting docker
+stop on stopped docker
+respawn
+
+# Start the process
+exec /usr/sbin/flocker-docker-plugin --logfile=/var/log/flocker/flocker-docker-plugin.log
```

Comment:
This log (anfd the systemd equivalent) will be automatically included in the ``flocker-diagnostics`` output because the file has the same prefix as the service....which is good. But if / when ``flocker-docker-plugin`` gets installed on acceptance nodes, the ``flocker-diagnostics`` test will fail because of unexpected extra files in the archive.
Something to consider in the following flocker-docker-plugin acceptance testing branch I suppose.